homeassistant/components/dyson/air_quality.py

```diff
@@ -0,0 +1,131 @@
+"""Support for Dyson Pure Cool Air Quality Sensors."""
+import logging
+
+from homeassistant.components.air_quality import AirQualityEntity, DOMAIN
+from . import DYSON_DEVICES
+
+DEPENDENCIES = ['dyson']
+
+ATTRIBUTION = 'Dyson purifier air quality sensor'
+
+_LOGGER = logging.getLogger(__name__)
+
+DYSON_AIQ_DEVICES = 'dyson_aiq_devices'
+
+ATTR_VOC = 'volatile_organic_compounds'
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Dyson Sensors."""
+    from libpurecool.dyson_pure_cool import DysonPureCool
+
+    if discovery_info is None:
+        return
+
+    _LOGGER.debug("Creating new Dyson air quality sensors")
```

Comment:
The core is already logging platform set up at debug level.

Reply:
Done.